wikimedia/wikipedia-ios
# Perform search on appearance of `InsertLinkViewController`

**Phabricator:** https://phabricator.wikimedia.org/T324862

### Notes
This resolves the issue where tapping link on an unlinked term in the section editor presented the insert link view, but did not immediately start populating search results for the selected term. Now, we perform a search on appearance of the `InsertLinkViewController`.

### Test Steps
1. Open an article and edit a section
2. Select a piece of unlinked text on the page and tap the link button in the toolbar
3. Confirm the Insert Link controller appears with the selected term in the search bar, and that a search is immediately performed to populate the results list
4. Confirm tapping a search result populates that link in the section editor
